### PR TITLE
Update Rabbit samples and snippets to RC1

### PR DESCRIPTION
**Pull request by WojcikMike**
cc: @Particular/docs-maintainers please merge when green

### CODE REVIEW

File: Snippets/Rabbit/Rabbit_4/Rabbit_4.csproj [hunk 1]
```diff
@@ -28,11 +28,12 @@
...
-    <Reference Include="NServiceBus.Transports.RabbitMQ">
-      <HintPath>..\..\..\packages\NServiceBus.RabbitMQ.4.0.0-beta0006\lib\net452\NServiceBus.Transports.RabbitMQ.dll</HintPath>
+    <Reference Include="NServiceBus.Transports.RabbitMQ, Version=4.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\NServiceBus.RabbitMQ.4.0.0-rc0001\lib\net452\NServiceBus.Transports.RabbitMQ.dll</HintPath>
+      <Private>True</Private>
     </Reference>
-    <Reference Include="RabbitMQ.Client, Version=3.6.5.0, Culture=neutral, PublicKeyToken=89e7d7c5feba84ce, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\RabbitMQ.Client.3.6.5\lib\net45\RabbitMQ.Client.dll</HintPath>
+    <Reference Include="RabbitMQ.Client, Version=4.0.0.0, Culture=neutral, PublicKeyToken=89e7d7c5feba84ce, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\RabbitMQ.Client.4.1.0-rc1\lib\net451\RabbitMQ.Client.dll</HintPath>
```
Comment (SimonCropp) [inline Snippets/Rabbit/Rabbit_4/Rabbit_4.csproj]: we should not be doing samples against an rc of the RabbitMQ.Client

Comment (adamralph) [inline Snippets/Rabbit/Rabbit_4/Rabbit_4.csproj]: @SimonCropp unfortunately our RC RabbitMQ transport depends on the RC of the RabbitMQ.Client. I.e. our RabbitMQ transport 4.0.0 cannnot go RTM until RabbitMQ.Client 4.1.0 goes RTM.

I think that our only option for now is to use the RabbitMQ.Client RC in our samples.

Comment (SimonCropp) [inline Snippets/Rabbit/Rabbit_4/Rabbit_4.csproj]: ahh. fair enough. i thought it was a nuget update mistake.

**Comment by SimonCropp** [comment]
so this broke the build due to a bad version or the NServiceBus.RabbitMQ packes on myget. spoke to @adamralph and he is going to push rc2

File: Snippets/Rabbit/Rabbit_4/Rabbit_4.csproj (hunk 1 above)
Comment (SimonCropp) [inline Snippets/Rabbit/Rabbit_4/Rabbit_4.csproj]: its out now https://www.nuget.org/packages/RabbitMQ.Client/4.1.0 

@adamralph can u change the nuget range when u do rc2?

Comment (adamralph) [inline Snippets/Rabbit/Rabbit_4/Rabbit_4.csproj]: We missed the chance by a matter of a minute :smile:

We'll go to RC3 then... :wink:

Comment (SimonCropp) [inline Snippets/Rabbit/Rabbit_4/Rabbit_4.csproj]: lol. sorry

**Comment by WojcikMike** [comment]
We pushed RC3 to reference RabbitMQ.Client 4.1

**Pull request merged**